packages/vendor-patches/tests/Finder/VendorFilesFinderSource/Vendor/some/package/src/PackageClass.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Symplify\VendorPatches\Tests\Finder\VendorFilesFinderSource;
+namespace Symplify\VendorPatches\Tests\Finder\VendorFilesFinderSource\Vendor\some\package\src;
 
 class PackageClass
 {
```

Comment:
This class should be kept as it is and excluded in `phpstan.neon`

Reply:
reverted.